tests/unit/common/test_table.py

```diff
@@ -1052,14 +1053,14 @@ def test_add_opt_column_mismatched_index_data(self):
     def test_add_opt_column_mismatched_col_cls(self):
         """Test that adding an optional column from __columns__ with non-matched table raises a warning."""
         table = SubTable(name='subtable', description='subtable description')
-        msg = ("Column 'col10' is predefined in SubTable with class=<class 'hdmf.common.table.VocabData'> "
+        msg = ("Column 'col10' is predefined in SubTable with class=<class 'hdmf.common.table.EnumData'> "
                "which does not match the entered col_cls "
                "argument. The predefined class spec will be ignored. "
                "Please ensure the new column complies with the spec. "
                "This will raise an error in a future version of HDMF.")
         with self.assertWarnsWith(UserWarning, msg):
-            table.add_column(name='col10', description='column #10', index=True)
-        self.assertEqual(table.col10.description, 'column #10')
+            table.add_column(name='col10', description='column #11', index=True)
+        self.assertEqual(table.col10.description, 'column #11')
```

Comment:
```suggestion
            table.add_column(name='col10', description='column #10', index=True)
        self.assertEqual(table.col10.description, 'column #10')
```
This change doesn't seem necessary.